### PR TITLE
LTI tool launch with Docker, Mysql support

### PR DESCRIPTION
**Pull request by pushyamig**
LTI launch with Docker setup connecting to MySQL Database. The LTI Library supports Multi-tenent so you can configure from any platform. I tried to add documentation and bare minimum. 

LTI registration from Django Admin view(LTI Tool)
Screenshot 2024-06-14 at 4 02 09 PM

LTI configuration from Canvas(Platform): look for the example setup in `lti-redirect-pu` and add the LTI tool to a course or account level. get the deployment ID and added to LTI tool

This is a bare minimum setup more work to come
1. CSP configuration, adding logging
2. Registing the LTI user to Django auth_user table mostly what MyLA does after user launches the tool
3. Make an API call to Maizey endpoint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+
+`python manage.py shell -c "from django.core.management.utils import get_random_secret_key; print(get_random_secret_key())"`
+
+## LTI install
+1. Need to run this command once in order for LTI to work
+```sh
+ docker exec -it clrt_web /bin/bash -c \
+  "python manage.py rotate_keys" 
+```  
+2. Create superuser via using `python maage.py createsuperuser', need to run a proxy like loophole or ngrok for LTI installation and login with that user.
```
Comment (zqian) [inline README.md]: A typo in `python maage.py createsuperuser`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+https://<url>/admin/
+4. Use `LTIRegistration` to configure an LTI tool
```
Comment (zqian) [inline README.md:17]: maybe add more description here for configure the LTI tool. 

can also mention this redirect tool uses CAI's django-lti package, and refer the django-lti documentation in this README file, too

Comment (pushyamig) [reply README.md:17]: Done

File: README.md (hunk 1 above)
Comment (pushyamig) [reply README.md]: Done

**Comment by pushyamig** [comment]
I will merge the PR if no more objection

**Review by zqian — APPROVED** [review]

**Comment by lsloan** [comment]
:shipit:

**Pull request merged**